vhost/src/vhost_user/gpu_backend_req.rs

```diff
@@ -24,7 +25,23 @@ impl BackendInternal {
         }
     }
 
-    fn send_message<T: ByteValued, V: ByteValued + Sized + Default + VhostUserMsgValidator>(
+    fn send_header<V: ByteValued + Sized + Default + VhostUserMsgValidator>(
+        &mut self,
+        request: GpuBackendReq,
+        fds: Option<&[RawFd]>,
+    ) -> vhost_user::Result<V> {
+        self.check_state()?;
+
+        //investigate what len should be
+        let _len = mem::size_of::<VirtioGpuRespDisplayInfo>();
+
+        let hdr = VhostUserGpuMsgHeader::new(request, 0, 0);
+        self.sock.send_header(&hdr, fds)?;
+
+        self.wait_for_ack(&hdr)
+    }
+
+    fn _send_message<T: ByteValued, V: ByteValued + Sized + Default + VhostUserMsgValidator>(
```

Comment:
Why does the name begin with an `_`, because it is unused?

Reply:
yes, unused for now, we will use it for getedid